ci/do_ci.sh

```diff
@@ -238,10 +238,21 @@ elif [[ "$CI_TARGET" == "bazel.distribution" ]]; then
 
   setup_clang_toolchain
 
-  # By default the packages will be signed by the first available key.
-  # If there is no key available, a throwaway key is created
-  # and the packages signed with it, for the purpose of testing only.
-  if ! gpg --list-secret-keys "*"; then
+  if [[ -n "$MAINTAINER_GPG_KEY" && -n "$MAINTAINER_GPG_KEY_PASSPHRASE" ]]; then
+      # Import the maintainer key
+      echo "$MAINTAINER_GPG_KEY_PASSPHRASE" | gpg --batch --yes --passphrase-fd 0 --import "$MAINTAINER_GPG_KEY"
```

Comment:
would be good to find a way of doing this without echoing the password (we are doing this already elsewhere unfortunately)